service/templates/patch_version.html

```diff
@@ -6,93 +6,111 @@
   var variants = {{.Variants}}
   var tasks = {{.Tasks}}
   var patch= {{.ProjectData.Patch}}
+  var commitQueuePosition = {{.CommitQueuePosition}}
   window.hasBanner = {{ne .Banner ""}}
 </script>
 {{end}}
 
 {{define "title"}}
+    {{ if patch.CommitQueue }}
```

Comment:
This needs to exactly match the name of the property that we are binding this template to, so it should be "Patch." Though is this needed? What if we use commit queue position > 0 to figure this out?

Reply:
That won't work because the patch could be no longer on the commit queue but still have been a commit queue patch (in which case we want to redirect to the version page). Johnathon's alias suggestion should work though.

Reply:
Also sorry, so it can't use the variable patch? Does it need to be the full .ProjectData.Patch then?

Reply:
Yep, patch is set in javascript but we haven't gotten there yet when this templating is run